metadata-service/war/src/main/resources/boot/retention.yaml

```diff
@@ -4,12 +4,12 @@
     retention:
       version:
         maxVersions: 5
-#- entity: "*"
-#  aspect: "*"
-#  config:
-#    retention:
-#      version:
-#        maxVersions: 20
+- entity: "*"
+  aspect: "*"
+  config:
+    retention:
+      version:
+        maxVersions: 20
```

Comment:
Can this be overridden by environment variables?

Reply:
No this is not supported currently.